tests/Agent/MultiverseTesting/ReportBuilder/DotNetFrameworkParser.cs

```diff
@@ -0,0 +1,80 @@
+﻿// Copyright 2020 New Relic, Inc. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+
+namespace ReportBuilder
+{
+    static public class DotNetMonikerNormalizer
+    {
+        private const string NetFramework = ".NET Framework 4.5.x to 4.8.x";
+        private const string NetStandard = ".NET Standard 2.0, 2.1";
+        private const string NetCore = ".NET Core 3.1 and .NET 5.0 to 7.0";
```

Comment:
Please add a comment indicating that this code is not currently used, and possibly comment the file out

Reply:
Removed.